src/statmodels.jl

```diff
@@ -233,3 +233,12 @@ function show(io::IO, ct::CoefTable)
         println(io)
     end
 end
+
+"""
+Exception to handle failure of convergence when fitting statistical models.
```

Comment:
Sorry not to have raised this before, but I think the description should avoid repeating "exception" and should be directed towards the user who sees this exception raised. See for example `?DomainError` or `?ArgumentError`.